### DISCUSSION TITLE
Docker-compose.yml doesn't work ? (Unsupported config option services,network,volumes)

### DISCUSSION THREAD

**Discussion by mythosmc** [firefly-iii/firefly-iii#9302] [Firefly III]
Anyone ever run into this? I used the file on the website - I did run the command as docker-compose rather than docker compose

docker:/home/firefly-iii # docker-compose -f docker-compose.yml up -d
ERROR: The Compose file './docker-compose.yml' is invalid because:
Unsupported config option for services: 'cron'
Unsupported config option for networks: 'firefly_iii'
Unsupported config option for volumes: 'firefly_iii_upload'

**Comment by JC5** [comment]
Is it because "docker compose" did not work either?

**Comment by jaw3l** [comment]
It seems there might be an indentation error. Could you please share your exact `docker-compose.yml` file?
